mathics_django/doc/django_doc.py

```diff
@@ -0,0 +1,658 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is there any difference between this and `mathics.doc.doc` ? Wouldn't be possible to just import the mathics (core) version?